### DISCUSSION TITLE
Linting

### DISCUSSION THREAD

**Discussion by AlexHedley** [Corofides/Movem#5] [Ideas]
Are we using any?

**Comment by Corofides** [comment]
I haven't really thought about it. Usually it tends to bug me but I can understand for consistency if you'd want to.

**Reply by AlexHedley** [reply]
Doesn't have to be too many rules but a few might be helpful.
Annoys me when it breaks a build for silly things like spacing.

Could always add a pre-commit hook?

**Comment by AlexHedley** [comment]
- [ktlint](https://github.com/pinterest/ktlint)
